icu4c/source/tools/icuexportdata/icuexportdata.cpp

```diff
@@ -47,68 +47,62 @@ int16_t DATAEXPORT_SCRIPT_X_WITH_INHERITED = 0x0800;
-      13180, 13181, 13182, 13183, 13280, 13281, 13282, 13283, 13284, 13285,
-      13286, 13287, 13288, 13289, 13290, 13291, 13292, 13293, 13294, 13295,
-      13296, 13297, 13298, 13299, 13300, 13301, 13302, 13303, 13304, 13305,
-      13306, 13307, 13308, 13309, 13310, 119648, 119649, 119650, 119651, 119652,
-      119653, 119654, 119655, 119656, 119657, 119658, 119659, 119660, 119661, 119662,
-      119663, 119664, 119665, 127568, 127569, 867, 868, 869, 870, 871,
-      872, 873, 874, 875, 876, 877, 878, 879, 7418, 7674,
-      66272, 66273, 66274, 66275, 66276, 66277, 66278, 66279, 66280, 66281,
-      66282, 66283, 66284, 66285, 66286, 66287, 66288, 66289, 66290, 66291,
-      66292, 66293, 66294, 66295, 66296, 66297, 66298, 66299, 1748, 64830,
-      64831, 1611, 1612, 1613, 1614, 1615, 1616, 1617, 1618, 1619,
-      1620, 1621, 1648, 65010, 65021, 7381, 7382, 7384, 7393, 7402,
-      7405, 7413, 7414, 43249, 12330, 12331, 12332, 12333, 43471, 65794,
-      65847, 65848, 65849, 65850, 65851, 65852, 65853, 65854, 65855, 1156,
-      1159, 11843, 42607, 1157, 1158, 1155, 7672, 7379, 7411, 7416,
-      7417, 7401, 7383, 7385, 7388, 7389, 7392, 43251, 4347, 3046,
-      3047, 3048, 3049, 3050, 3051, 3052, 3053, 3054, 3055, 3056,
-      3057, 3058, 3059, 70401, 70403, 70459, 70460, 73680, 73681, 73683,
-      2790, 2791, 2792, 2793, 2794, 2795, 2796, 2797, 2798, 2799,
-      2662, 2663, 2664, 2665, 2666, 2667, 2668, 2669, 2670, 2671,
-      42752, 42753, 42754, 42755, 42756, 42757, 42758, 42759, 12337, 12338,
-      12339, 12340, 12341, 12441, 12442, 12443, 12444, 12448, 12540, 65392,
-      65438, 65439, 3302, 3303, 3304, 3305, 3306, 3307, 3308, 3309,
-      3310, 3311, 8239, 68338, 6146, 6147, 6149, 1564, 1632, 1633,
-      1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 2534, 2535,
-      2536, 2537, 2538, 2539, 2540, 2541, 2542, 2543, 4160, 4161,
-      4162, 4163, 4164, 4165, 4166, 4167, 4168, 4169, 65792, 65793,
-      65799, 65800, 65801, 65802, 65803, 65804, 65805, 65806, 65807, 65808,
-      65809, 65810, 65811, 65812, 65813, 65814, 65815, 65816, 65817, 65818,
-      65819, 65820, 65821, 65822, 65823, 65824, 65825, 65826, 65827, 65828,
-      65829, 65830, 65831, 65832, 65833, 65834, 65835, 65836, 65837, 65838,
-      65839, 65840, 65841, 65842, 65843, 7412, 8432, 12348, 12349, 43310,
-      7376, 7378, 5941, 5942, 2406, 2407, 2408, 2409, 2410, 2411,
-      2412, 2413, 2414, 2415, 12291, 12307, 12316, 12317, 12318, 12319,
-      12336, 12343, 65093, 65094, 1548, 1563, 12289, 12290, 12296, 12297,
-      12298, 12299, 12300, 12301, 12302, 12303, 12304, 12305, 12308, 12309,
-      12310, 12311, 12312, 12313, 12314, 12315, 12539, 65377, 65378, 65379,
-      65380, 65381, 7386, 1567, 7410, 1600, 43062, 43063, 43064, 43065,
-      2386, 2385, 43059, 43060, 43061, 43056, 43057, 43058, 2404, 2405
-    };
+     0x00b7, 0x02bc, 0x02c7, 0x2c9, 0x2ca, 0x02cd, 0x02d7, 0x02d9, 0x0300, 0x0301, 0x0302, 0x0303,
```

Comment:
Your script must have a bug, in https://www.unicode.org/Public/16.0.0/ucd/ScriptExtensions.txt 
```
02C7          ; Bopo Latn                      # Lm      CARON
02C9..02CB    ; Bopo Latn                      # Lm  [3] MODIFIER LETTER MACRON..MODIFIER LETTER GRAVE ACCENT
02CD          ; Latn Lisu                      # Lm      MODIFIER LETTER LOW MACRON
```
but in your output, we see only 
```
0x02c7,  0x2c9, 0x2ca, 0x02cd,
```
and there are no 0x2cb 
also notice it show " 0x2c9, 0x2ca," instead of " 0x02c9, 0x02ca," (missing 0 after x)

Reply:
notice https://www.w3schools.com/python/ref_func_range.asp
range(start, stop, step)

stop | Required. An integer number specifying at which position to stop (not included).
-- | --



the stop is not included in the range() in python

Reply:
Ah, good point, added 1 to it